vcl_templates/assets.vcl.erb

```diff
@@ -153,6 +153,11 @@ sub vcl_recv {
      error 801 "Force SSL";
   }
 
+  # Serve and log an empty response for analytics purposes
+  if (req.url == "/static/a") {
```

Comment:
does the URL receive parameters? I think req.url might include them, maybe ~ is safer